src/main/java/classify/data/DataWriter.java

```diff
@@ -62,4 +62,20 @@ public static void createParentFileFolder (String parentPath) {
             DataUI.printCreateFailure();
         }
     }
+
+    //@@author alalal47
+    /**
+     * Used to delete the contents of the file containing student information.
+     * Only used in the case of corruption that cannot be resolved by the user.
+     */
+    public static void emptyDataFile(String folderPath, String filePath) {
+        try {
+            createParentFileFolder(folderPath);
+            FileWriter overwriteBlank = new FileWriter(filePath);
+            overwriteBlank.write("");
+            DataUI.printFileDeletionCompleteMessage();
+        } catch (IOException e) {
+            DataUI.printFileIOError();
+        }
+    }
```

Comment:
Nice addition to allow for file wipe should the file be corrupted and the user chooses to do so